### PR TITLE
Update docs to synch with cookiecutter updates

### LINKED ISSUE #3081: update docs on master for 1.9 release checklist
This is just a checklist to help me keep track of all the docs changes per updates in the cookiecutters.

See #3042, #3048, #3052.

Modify the `src` files:
- [x] #3042 `docs/` search for `config.include('pyramid_tm')`, replace with:
  ```
      config.include('pyramid_tm')
      config.include('pyramid_retry')
  ```
- [x] #3059 `docs/` search for `config = Configurator(settings=settings)`, replace with:
  ```
      config = Configurator(settings=settings)
      settings = config.get_settings()
      settings['tm.manager_hook'] = 'pyramid_tm.explicit_manager'
  ```
- [x] #3059 `docs/` search for `config = Configurator(root_factory=root_factory, settings=settings)`, replace with:
  ```
      config = Configurator(root_factory=root_factory, settings=settings)
      settings = config.get_settings()
      settings['tm.manager_hook'] = 'pyramid_tm.explicit_manager'
  ```
- [x] #3059 `docs/` search for `settings = config.get_settings()`, for only those that are not duplicates of previous two items, replace with:
  ```
      settings = config.get_settings()
      settings['tm.manager_hook'] = 'pyramid_tm.explicit_manager'
  ```

Modify the `.rst` files:
- [x] `docs/` regex for `literalinclude.*setup.py` found 21 matches in 16 files
- [x] `docs/tutorials` regex for `literalinclude.*__init__.py` found 20 matches in 5 files
- [x] `docs/` search for:
  ```
  requires = [
    'pyramid
  ```

### PR DESCRIPTION
**Pull request by stevepiercy**
Closes #3081.
Refs #3042, #3048, #3052.

### CODE REVIEW

File: docs/tutorials/wiki2/src/models/tutorial/__init__.py [hunk 1]
```diff
@@ -5,8 +5,6 @@ def main(global_config, **settings):
     """ This function returns a Pyramid WSGI application.
     """
     config = Configurator(settings=settings)
-    settings = config.get_settings()
```
Comment (mmerickel) [inline docs/tutorials/wiki2/src/models/tutorial/__init__.py]: Why is this removed?

Comment (stevepiercy) [inline docs/tutorials/wiki2/src/models/tutorial/__init__.py]: Heh, yeah, this made me do a double-take, too. Notice the directory structure. These lines appear one directory deeper in the hierarchy within `project_name/models/__init__.py`, at `docs/tutorials/wiki2/src/*/tutorial/models/__init__.py` for each of the seven steps of the tutorial and the alchemy cookiecutter at https://github.com/Pylons/pyramid-cookiecutter-alchemy/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/%7B%7Bcookiecutter.repo_name%7D%7D/models/__init__.py#L59-L60. I assumed it was an accidental addition at some point.

**Comment by mmerickel** [comment]
Yeoman's work - thank you @stevepiercy !
